services/frontend-service/src/ui/components/button/button.tsx

```diff
@@ -14,14 +14,16 @@ You should have received a copy of the GNU General Public License
 along with kuberpult.  If not, see <http://www.gnu.org/licenses/>.
 
 Copyright 2021 freiheit.com*/
-import { useRef, useEffect } from 'react';
+import { useRef, useEffect, useCallback } from 'react';
 import classNames from 'classnames';
 import { MDCRipple } from '@material/ripple';
 
-export const Button = (props: { className?: string; label?: string }) => {
+export const Button = (props: { className?: string; label?: string; icon?: string; clickFunction?: any }) => {
```

Comment:
nit: please name this also `onClick` instead of `clickFunction`

Reply:
Ffixed!